taskcluster/focus_android_taskgraph/target_tasks.py

```diff
@@ -11,8 +11,16 @@ def target_tasks_default(full_task_graph, parameters, graph_config):
     """Target the tasks which have indicated they should be run on this project
     via the `run_on_projects` attributes."""
 
-    filter = filter_for_tasks_for
-    return [l for l, t in full_task_graph.tasks.items() if filter_for_tasks_for(t, parameters)]
+    # Temporary until we have official beta support
+    def releases(task, parameters):
+        return task.attributes.get("release-type", "") == 'release'
+
+    _filter = filter_for_tasks_for
+
+    if parameters["tasks_for"] == "github-release":
+        _filter = releases
+
+    return [l for l, t in full_task_graph.tasks.items() if _filter(t, parameters)]
 
 
 @_target_task('release')
```

Comment:
I didn't use the target_task logic yet because in Fenix the `target-tasks-for` to hit this custom release target task is set in the custom parameters.py since it doesn't look like it'll be needed once we switch to shipit releases as part of the next segment of beta/release work.

Reply:
Yeah, I don't think we'll need to pay attention `github-release` at all after that work is done (because Ship It should always be used to kick off releases).